### ISSUE TITLE
Sprint 2 – Scrum Meeting (9-12-2016) 

### ISSUE THREAD

**Issue by huylin25** [mewind/RSDSupermarketPOS#9]
Hi guys, here is our meeting again. So how's the progression of Add to Cart function?

**Comment by mewind** [comment]
I think we are almost done with it, will upload the code later. We had added a "Add to Cart" from that will be called everytime the user double-clicking the product they are desired to buy

**Comment by lkc442012** [comment]
Ya, we had successfully apply our algorithm where we just need to add the product user wants to buy into a Sales database, so that the GridView in Payment tab will have the products that are in the cart.

**Comment by huylin25** [comment]
Seems things getter better after the discussion yesterday! So will you guys upload the code tonight?

**Comment by mewind** [comment]
I think if possible will be tonight but latest might be tomorrow. Anyway, we are now finish doing one of the most challenging part in this module, feeling so much relieved after developing that.

**Comment by huylin25** [comment]
What do you guys plan to do next?

**Comment by lkc442012** [comment]
Will be adding some minor part to enhance the form's function. I think maybe will be the Search function in each of the category forms

**Comment by juzplayplay** [comment]
I think for me I'll try to do the validation part where the system will crosscheck the stock quantity of the product that user want to buy. Will try to include an error message if the quantity is stock is insufficient to let user decide whether to reduce the quantity or not to buy the product.

**Comment by huylin25** [comment]
This will be one of the crucial part also, because this might make the system to be more considerate and fault tolerant, it allows the user to recover from mistakes when abnormal activity is detected. Good job Raymond!

**Comment by huylin25** [comment]
For KarCheng and Sin Nee, you guys are planning to include the Search function in every single Category form?

**Comment by mewind** [comment]
Yes, I think we will try to implement this in every category form so that user can filter the search results and make the system easy to use.

**Comment by huylin25** [comment]
Great! So, I think we can actually start working on it! You guys did a excellent job, the development tasks are completing way faster compared to my expected development time!

**Comment by lkc442012** [comment]
Haha, we are just trying our best to complete the job. Thanks HuyLin for being so responsible in guiding us and so systematic in assigning the tasks. Without you the development of module may not have such a well progress.

**Comment by mewind** [comment]
Ya, KarCheng is right, thanks HuyLin!

**Comment by huylin25** [comment]
You're welcome ! That's my responsibility as a Scrum Master! So we can dismiss the meeting now and start doing the work , just dont forget to find me if you met any difficulty during development.

**Comment by juzplayplay** [comment]
Sure, thanks HuyLin.
